AnimateSvg.js

```diff
@@ -46,69 +90,28 @@ export default function AnimateSvg(props) {
         useNativeDriver: true,
       }),
     ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+  };
+
+  // LogoSvg fades out first time
+  if (isShakeTriggered && !hasInitialTransitionRun && !isLogoAnimationRunning) {
+    setIsLogoAnimationRunning(true);
+    logoAnimationOutInitialization();
     // LogoSvg fades in and then back out in sequence
-  } else if (toggle && hasToggledBefore && !isRegistered) {
-    setIsRegistered(true);
-    Animated.sequence([
-      Animated.parallel([
-        Animated.timing(fadeAnim, {
-          toValue: 1,
-          duration: SVG_IN_DURATION,
-          delay: SVG_IN_DELAY,
-          useNativeDriver: true,
-        }),
-        Animated.timing(spinValue, {
-          toValue: 0,
-          duration: SVG_IN_DURATION,
-          delay: SVG_IN_DELAY,
-          useNativeDriver: true,
-        }),
-        Animated.timing(sizeAnim, {
-          toValue: 1,
-          duration: SVG_IN_DURATION,
-          delay: SVG_IN_DELAY,
-          useNativeDriver: true,
-        }),
-      ]),
-      Animated.parallel([
-        Animated.timing(fadeAnim, {
-          toValue: 0,
-          duration: SVG_OUT_DURATION,
-          useNativeDriver: true,
-        }),
-        Animated.timing(spinValue, {
-          toValue: 1,
-          duration: SVG_OUT_DURATION,
-          useNativeDriver: true,
-        }),
-        Animated.timing(sizeAnim, {
-          toValue: 0.5,
-          duration: SVG_OUT_DURATION,
-          useNativeDriver: true,
-        }),
-      ]),
-    ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(QUOTES_OUT_DURATION, SVG_IN_DURATION) + SVG_IN_DELAY + Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+  } else if (isShakeTriggered && hasInitialTransitionRun && !isLogoAnimationRunning) {
+    setIsLogoAnimationRunning(true);
+    logoAnimationIn_CallbackOut();
   }
 
   return (
     <Animated.View style={{
-      ...props.style,
```

Comment:
No longer utilizing `props` object in component, nor passing styles through props.

Reply:
remove this comment